WordPress/src/main/java/org/wordpress/android/ui/reader/discover/interests/ReaderInterestsViewModel.kt

```diff
@@ -90,11 +88,34 @@ class ReaderInterestsViewModel @Inject constructor(
         _uiState.value = uiState
     }
 
-    data class UiState(
+    sealed class UiState(
         val interestsUiState: List<InterestUiState>,
         val interests: ReaderTagList,
```

Comment:
`InterestsUiState`, `Interests` are only needed when data is loaded. Will refactor to not include them for other states in the next PR that handles failed state.